packages/python3.rb

```diff
@@ -90,8 +105,12 @@ def self.install
 
   def self.postinstall
     puts
-    puts "Upgrading pip...".lightblue
+    puts 'Upgrading pip...'.lightblue
     system "curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && LD_LIBRARY_PATH=#{CREW_LIB_PREFIX} python3 get-pip.py -U"
-    puts
+    puts 'Setting Python3 as default python...'.lightblue
+    FileUtils.rm "#{CREW_PREFIX}/bin/python"
+    FileUtils.ln_s "#{CREW_PREFIX}/bin/python3", "#{CREW_PREFIX}/bin/python"
+    FileUtils.rm "#{CREW_PREFIX}/bin/pip"
+    FileUtils.cp "#{CREW_PREFIX}/bin/pip3", "#{CREW_PREFIX}/bin/pip"
```

Comment:
Shouldn't need the `FileUtils.rm ...` lines with `FileUtils.ln_sf ...` and `FileUtils.cp_f ...`.

Reply:
Should we be using `FileUtils.ln_sf "#{CREW_PREFIX}/bin/pip3", "#{CREW_PREFIX}/bin/pip"` instead?

Reply:
Upon install, poip3 and pip are tiny script files, not symlinks.

Example from ubuntu:
```
#!/usr/bin/python3
# EASY-INSTALL-ENTRY-SCRIPT: 'pip==20.1.1','console_scripts','pip'
__requires__ = 'pip==20.1.1'
import re
import sys
from pkg_resources import load_entry_point

if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
    sys.exit(
        load_entry_point('pip==20.1.1', 'console_scripts', 'pip')()
    )
```

Seems crazy to use a symlink for those, no?

Reply:
Well, the idea is to not have duplicate files.  A symlink will still work, no?

Reply:
I assume a symlink will still work? Since it is different from a normal python install though we can't be sure if something might break...